modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -108,23 +133,19 @@ curl "http://localhost/myResource" -u 1234:abcd
 
 Example DataWeave 2.0 expression to be used when configuring the policy:
 
-No DataWeave 2.0 expression needed to be specified when the policy is configured to use the Basic Authentication as the origin of the credentials.
+To use the Basic Authentication as the origin of the credentials, no DataWeave 2.0 expression needs to be specified when configuring the policy.
 
-== WWW Authenticate HTTP header
-When a HTTP request is performed against a protected resource, where the Client ID enforcement policy is applied, and client application credentials are invalid or unauthorized, the HTTP response includes a WWW Authenticate header with the following values:
 
-[width="100%", cols="2,70"]
-|===
->s| Custom mode | Header 'WWW-Authenticate'='Client-ID-Enforcement'
->s| Basic Auth mode | Header 'WWW-Authenticate'='Basic realm="mule-realm"
-|===
+=== Configure API Specifications
+//What to add here? -- Eva (this is not needed?)
```

Comment:
It's already covered earlier in the doc, I would leave it for now and marked it as one of the improvements areas in the documentation: how to update API spec once a policy is applied.

Reply:
Done! Thanks!